### PR TITLE
Create member profile for Bariq Nurlis

### PR DESCRIPTION
**Pull request by sudo-NithishKarthik**
**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit d69101d5c540d4014b57d48dd7a391df003dc310 at: https://61346c36a4f0215c8a43afdf--layer5ng.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 851e44fd0f83fe8f9655f946bc6ea5c00380b01b at: https://61347938aa4ceca552cecadf--layer5ng.netlify.app

**Review by warunicorn19 — APPROVED** [review]
CITY😄

**Review by warunicorn19 — APPROVED** [review]

File: src/collections/members/bariq-nurlis/index.mdx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+name: Bariq Nurlis
+position: Contributor
+image_path: ./bariq-nurlis.jpeg
+github: bariqhibat
+linkedin: bariqhibat
+location: Singapore/Indonesia
+bio: Bariq is an undergraduate student who currently studying engineering in NTU Singapore. He is a passionate maker who likes to build things and watch them benefitting people's lives. He's excited to contribute and learn in the open source community.
+badges:
```
Comment (Nikhil-Ladha) [inline src/collections/members/bariq-nurlis/index.mdx:9]: Meshery?

Comment (Nikhil-Ladha) [reply src/collections/members/bariq-nurlis/index.mdx:9]: cc @leecalcote

Comment (leecalcote) [reply src/collections/members/bariq-nurlis/index.mdx:9]: Yes, "meshery" make sense. 👍

Comment (leecalcote) [inline src/collections/members/bariq-nurlis/index.mdx:9]:
```suggestion
badges: meshery
```

Comment (bariqhibat) [reply src/collections/members/bariq-nurlis/index.mdx]: Yes, I think it'd be good to put Layer5/Meshery instead 

// @sudo-NithishKarthik

File: src/collections/members/bariq-nurlis/index.mdx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+name: Bariq Nurlis
+position: Contributor
+image_path: ./bariq-nurlis.jpeg
+github: bariqhibat
+linkedin: bariqhibat
+location: Singapore/Indonesia
+bio: Bariq is an undergraduate student who currently studying engineering in NTU Singapore. He is a passionate maker who likes to build things and watch them benefitting people's lives. He's excited to contribute and learn in the open source community.
+badges: meshery
```
Comment (Nikhil-Ladha) [inline src/collections/members/bariq-nurlis/index.mdx:9]: It should be like:
```
badges:
 - meshery
```
For example this [profile](https://raw.githubusercontent.com/layer5io/layer5/master/src/collections/members/aditi-ahuja/index.mdx)
@leecalcote

Comment (warunicorn19) [reply src/collections/members/bariq-nurlis/index.mdx:9]:
```suggestion
badges: 
  - meshery
```

**Comment by l5io** [comment]
🚀 Preview for commit f882515e4df86b235c361fefd0e4e25cf881e93b at: https://61376aaf8a16cb925cb217c0--layer5ng.netlify.app

**Review by Tanuj22 — APPROVED** [review]

**Pull request merged**